src/DssSpell.sol

```diff
@@ -74,190 +120,324 @@ contract DssSpellAction is DssAction {
+    uint256 internal constant THREE_PT_ONE_NINE_PCT_RATE  = 1000000000995743377573746041;
+    uint256 internal constant THREE_PT_FOUR_FOUR_PCT_RATE = 1000000001072474267302354182;
+    uint256 internal constant THREE_PT_NINE_FOUR_PCT_RATE = 1000000001225381266358479708;
+    uint256 internal constant FIVE_PT_FOUR_FOUR_PCT_RATE  = 1000000001679727448331902751;
+    uint256 internal constant FIVE_PT_SIX_NINE_PCT_RATE   = 1000000001754822903403114680;
+    uint256 internal constant SIX_PT_ONE_NINE_PCT_RATE    = 1000000001904482384730282575;
+
+    // 2023-06-26 00:00:00 UTC
+    uint256 internal constant JUN_26_2023                 = 1687737600;
+    // 2023-07-01 00:00:00 UTC
+    uint256 internal constant JUL_01_2023                 = 1688169600;
+    // 2024-06-30 23:59:59 UTC
+    uint256 internal constant JUN_30_2024                 = 1719791999;
+    // 2024-12-31 23:59:59 UTC
+    uint256 internal constant DEC_31_2024                 = 1735689599;
+
+    // Delegates
+    address internal constant DEFENSOR                    = 0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9;
+    address internal constant BONAPUBLICA                 = 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3;
+    address internal constant QGOV                        = 0xB0524D8707F76c681901b782372EbeD2d4bA28a6;
+    address internal constant TRUENAME                    = 0x612F7924c367575a0Edf21333D96b15F1B345A5d;
+    address internal constant UPMAKER                     = 0xbB819DF169670DC71A16F58F55956FE642cc6BcD;
+    address internal constant VIGILANT                    = 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61;
+    address internal constant WBC                         = 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47;
+    address internal constant PBG                         = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
+    address internal constant BANDHAR                     = 0xE83B6a503A94a5b764CCF00667689B3a522ABc21;
+    address internal constant LIBERTAS                    = 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f;
+    address internal constant PALC                        = 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A;
+    address internal constant HARMONY                     = 0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2;
+    address internal constant VOTEWIZARD                  = 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1;
+    address internal constant NAVIGATOR                   = 0x11406a9CC2e37425F15f920F494A51133ac93072;
+
+    // Ecosystem actors
+    address internal constant DECO_WALLET                 = 0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7;
+    address internal constant DUX_WALLET                  = 0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad;
+    address internal constant CHRONICLE_LABS              = 0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f;
+    address internal constant JETSTREAM                   = 0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962;
+
+    GemLike internal immutable MKR                        = GemLike(DssExecLib.mkr());
```

Comment:
```suggestion
    GemLike  internal immutable MKR                       = GemLike(DssExecLib.mkr());
```
Nit

Reply:
Actually disregard this as the VestLike -> DssVestLike change will make this above suggestion incorrect again. Fix spacing if you want, not a real issue.